dss.go

```diff
@@ -0,0 +1,119 @@
+package globalsign
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/hex"
+	"strings"
+	"time"
+)
+
+// DSSService availables GlobalSign Digital Signing Service.
```

Comment:
availables?  Not sure what that is supposed to mean.. Maybe "implements" or "defines what is needed for..."?

Reply:
Updated at 6fa006e290519f3ae08c512c53671a7b8ce4e71f